src/Umbraco.Infrastructure/Persistence/NPocoSqlExtensions.cs

```diff
@@ -113,10 +119,12 @@ public static Sql<ISqlContext> WhereIn<TDto>(this Sql<ISqlContext> sql, Expressi
         }
 
 
-        public static Sql<ISqlContext> WhereLike<TDto>(this Sql<ISqlContext> sql, Expression<Func<TDto, object?>> fieldSelector, Sql<ISqlContext>? valuesSql)
+        public static Sql<ISqlContext> WhereLike<TDto>(this Sql<ISqlContext> sql, Expression<Func<TDto, object?>> fieldSelector, Sql<ISqlContext>? valuesSql, string concatDefault = "")
```

Comment:
I know we don't have them for all methods already, but I think this would benefit from an XML header comment, explaining what each parameter is for (in particular the new one added).

Reply:
Added.